management-server/src/main/kotlin/ut/isep/management/service/converter/applicant/ApplicantCreateConverter.kt

```diff
@@ -0,0 +1,20 @@
+package ut.isep.management.service.converter.applicant
+
+import dto.applicant.ApplicantCreateDTO
+import org.springframework.stereotype.Component
+import ut.isep.management.model.entity.Applicant
+import ut.isep.management.service.converter.CreateConverter
+
+@Component
+class ApplicantCreateConverter : CreateConverter<Applicant, ApplicantCreateDTO> {
+
+    override fun fromDTO(createDTO: ApplicantCreateDTO): Applicant {
+        return Applicant(
+            id = 0, // Or some default
+            name = createDTO.name,
+            status = createDTO.status,
```

Comment:
Should this also not have email as an attribute?

Reply:
Addressed in https://github.com/Alakind/ISEP/commit/1f329faaf32e1020047d9125ff76fcad7bc8190e